[DOC] Restructure user documentation (#945)
saros-project/saros: docs/documentation/how-tos/manage-xmpp-accounts.md

- Select option `Add Account...`.
- Follow the steps of the wizard.

*Alternatively* via preferences:

- Open the [Saros account preferences](#saros-account-preferences)
- Click on `Add Account`.
- Follow the steps of the wizard.

## Remove an Account

- Open the [Saros account preferences](#saros-account-preferences)
- Select the account you want to remove in the `XMPP Accounts` list.
- Click on `Remove Account`.

## Modify an Account

- Open the [Saros account preferences](#saros-account-preferences)
- Select the account you want to modify in the `XMPP Accounts` list.
- Click on `Edit Account`.
- Change the JID or the password (if you want to change network specific information open the `Advanced Options`)
- Click on `Finish`

## Change the Default Account

- Open the [Saros account preferences](#saros-account-preferences)
- Select the account you want to define as default account.
- Click on `Activate Account`

{% endcapture eclipse %}
{% capture intellij %}
## Add an Account

- In the top bar of the Saros view, click the "Connect" button (![connect icon](images/icons/connect.png)).
- Choose `Add account...` from the pop-up menu.

{% alert warning %}
## Note

> As mentioned in the section [missing features](/releases/saros-i_0.2.2.html#missing-features), Saros for IntelliJ **does currently not support the creation, management or deletion of XMPP accounts**. As only people on your friends list can be invited to join your Saros session, you will have to create an account and add friends to your friends list through a different client. Any XMPP client (including Saros for Eclipse) can be used for this purpose.

Reviewer: Shame on me for forgetting the oxford coma in the original :wink:. 

Also, the part about having to add contacts through other clients is not true anymore. This feature has been added with #785. There also already is a section on how to add an contacts in Saros/I in the old and new getting started page, so it seems like I just forgot to remove this section.

```suggestion
As mentioned in the section [missing features](/releases/saros-i_0.2.2.html#missing-features), Saros for IntelliJ **does currently not support the creation, management, or deletion of XMPP accounts**.
```

> If you accidentally made a typo while entering your username or password, the created account entry can also only be changed or deleted through Saros for Eclipse (or by deleting the account store `~/.saros/config.dat`). Saros for Eclipse does not permit the deletion of the currently chosen account. If you only added one account and would like to remove it, you will have to add a second account (for example with random values) and choose this new account as the default. You can then delete the first account, add a new account with the right values, choose it as the correct default and delete the temporary account entry.

Reviewer: The section about Saros/E not allowing to delete the active account is no longer true.

```suggestion
If you accidentally made a typo while entering your username or password, the created account entry can currently only be changed or deleted through Saros for Eclipse.

Alternatively, you could delete the Saros account store `~/.saros/config.dat`. **Note that this will remove all stored XMPP accounts from the local configuration.** They can be re-added after restarting Saros.
```

- Select option `Add Account...`.
- Follow the steps of the wizard.

*Alternatively* via menu bar:

- Open the "Saros" drop-down menu in the menu bar.
- Select option `Add Account...`.
- Follow the steps of the wizard.

*Alternatively* via preferences:

- Open the [Saros account preferences](#saros-account-preferences)
- Click on `Add Account`.
- Follow the steps of the wizard.

## Remove an Account

- Open the [Saros account preferences](#saros-account-preferences)
- Select the account you want to remove in the `XMPP Accounts` list.
- Click on `Remove Account`.

## Modify an Account

- Open the [Saros account preferences](#saros-account-preferences)
- Select the account you want to modify in the `XMPP Accounts` list.
- Click on `Edit Account`.
- Change the JID or the password. (If you want to change network specific information, open the `Advanced Options`.)
- Click on `Finish`.

## Change the Default Account

- Open the [Saros account preferences](#saros-account-preferences).
- Select the account you want to define as default account.
- Click on `Activate Account`.

{% endcapture eclipse %}
{% capture intellij %}
## Add an Account

> - In the top bar of the Saros view, click the "Connect" button (![connect icon](images/icons/connect.png)).

Reviewer: ```suggestion
- In the top bar of the Saros view, click the "Connect" button (![connect icon](../images/icons/connect.png)).
```
